command/policy_write.go

```diff
@@ -127,3 +130,12 @@ func (c *PolicyWriteCommand) Run(args []string) int {
 	c.UI.Output(fmt.Sprintf("Success! Uploaded policy: %s", name))
 	return 0
 }
+
+func warnOnUpperCase(ui cli.Ui, s string) {
+	for _, r := range s {
+		if unicode.IsUpper(r) && unicode.IsLetter(r) {
```

Comment:
Super nit-picky/ curious question. If `go` supports short circuiting, should this check the Letter first and then whether or not it's uppercase?

Reply:
Or you may just need the `Upper` check b/c it seems like it also checks that it's a letter? idk

Reply:
I think you're totally right, but resolving this conversation as I went with a different approach that doesn't iterate over the letters, after discussing with @VinnyHC .